library/std/src/os/freebsd/fs.rs

```diff
@@ -136,6 +141,7 @@ impl MetadataExt for Metadata {
     fn st_flags(&self) -> u32 {
         self.as_inner().as_inner().st_flags as u32
     }
+    #[cfg(not(freebsd12))]
     fn st_lspare(&self) -> u32 {
```

Comment:
huh, this function outright missing doesn't cause errors elsewhere in if the flag is active?

Reply:
Definitely not in Rust itself. Maaaaaaybe there's some user code out there using it, but I wouldn't be surprised if nobody ever touched this obscure field on this deprecated way to access the raw stat data.. Okay I guess it would be better to panic instead